CREATE_RELEASE.md

```diff
@@ -25,24 +25,42 @@ Or if you want to use the conventional commits to determine next version automat
 
-## 3. Hotfix releases
+While we are under `v1` stage, BREAKING CHANGES are incrementing the minor version, and the rest of the changes
+are incrementing the patch version. After we reach `v1`, BREAKING CHANGES will increment the major version, and
+the rest of the changes will increment the minor version or the patch, depending on the type of the change.
+
+Let's say the latest release was `0.40.0`, and you merged a PR with the `fix` commit type. In this case, the next
+version will be `0.40.1-rc.0` and ever subsequent merge will increment the `rc` version, until the next `Stable`
+release is created, after which the version without the `rc` will be created. If you merge a PR with the `BREAKING CHANGE`,
+then the next version will be `0.41.0-rc.0`, and the subsequent merges will increment the `rc` version, even if the PRs
+contain `BREAKING CHANGE` again.
+
+## Hotfix releases
 
 `Hotfix` releases are also created automatically, but the difference from `Stable` is the source of the release.
-When you want to create a `Hotfix`, you need to checkout the tag, which will be enriched with the hotfix content.
-For example if your latest version is `v0.40.1` and after that release there have been numerous releases with the
-`v0.40.2-rc.*` tags, and you want to create a hotfix, which fixes `v0.40.1` with additional commit, but you do not
-want to include all the changes from `v0.40.2-rc.*` releases, then you need to checkout the `v0.40.1` tag, and
-cherry-pick a commit with the hotfix. Then you need to type in `npm run hotfix-release` and the release process will
-start. This is a fully automatic process, and the version of the release is determined using the conventional commits.
-In this situation, the version of the release will be `v0.40.2` and since the `v0.40.2` is newer than the main branch
-version, `v0.40.2-rc.*`, the main branch will be updated to `v0.40.2` as well, but only with the version indicator,
-not the change itself. This ensures that after the next rc release on main, the versions will not collide. Also,
-this given `hotfix` will be marked as `latest` on npm.
+
+Let's go through the process of creating a hotfix:
+Imagine that the latest release was 0.40.0, and you want to add one patch to that release, but you do not want to
+include all the changes from the 0.40.1 release to the latest RC release, which is 0.40.1-rc.\*. In this case you need
+to do the following:
+
+-   `git checkout v0.40.0`
+-   `git cherry-pick <commit-hash>` or make changes you want to make and commit them
+-   `npm run hotfix-release`
+
+After this, lerna determines that the next version should be `0.40.1`, and CI will create a release from the HEAD.
+CI also finds out that the `0.40.1` is newer than the latest RC release, which is `0.40.1-rc.*`, and updates the main
+branch version to `0.40.1`. Also it must be noted that the npm will mark this release as the `latest`.
```

Comment:
I think this isn't always true, not the case for the ng15 hotfixes I've been doing

Reply:
Basically after lerna step, it compares whether the given result is bigger than the latest `latest` and chooses bigger version for the `latest` candidate